test/integration/kubernetes_deploy_test.rb

```diff
@@ -917,6 +919,33 @@ def test_cronjobs_can_be_deployed
     cronjobs.assert_cronjob_present("my-cronjob")
   end
 
+  def test_jobs_can_be_successful
+    assert_deploy_success(deploy_fixtures("hello-cloud", subset: ["job.yml"]))
+    assert_logs_match_all([
+      "Deploying Job/hello-job (timeout: 600s)",
```

Comment:
I'd suggest adding this assertion to `full_hello...` instead, since it's the only thing that test doesn't already cover. Generally speaking I'd consider that sufficient for asserting the basic success case.